xc7/utils/prjxray_create_edges.py

```diff
@@ -158,7 +158,8 @@ def find_pip(tile_type, pip):
 FROM
   pip_in_tile
 WHERE
-  name = ?
+  is_directional = 1 AND is_pseudo = 0
```

Comment:
Rather than filter here, please output is_directional and preserve the "# TODO: Handle bidirectional pips?"

Reply:
I removed this condition from that query. It was not needed here because `make_connections` is always called for non-pseudo and directional pips. The comment about supporting bi-pips is a few lines above the call.